app/src/main/java/mikekang47/spring/project/application/MovieService.java

```diff
@@ -0,0 +1,17 @@
+package mikekang47.spring.project.application;
+
+import mikekang47.spring.project.domain.MovieRepository;
+import mikekang47.spring.project.models.Movie;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
```

Comment:
이런 부분에서 조금씩 더 신경써주세요.
```suggestion
import java.util.List;

@Service
```